src/features/sync.ts

```diff
@@ -13,13 +14,16 @@ const logUpdate = (action: "created" | "updated" | "deleted", event: CalendarEve
   logger.info("event", [
     `${action.toUpperCase()}`,
     `- title: ${event.summary}`,
-    `- date: ${day.utc(event.start?.dateTime).format("LLLL")} (UTC)`,
+    `- date: ${day.utc(event.start?.dateTime ?? event.start?.date).format("LLLL")} (UTC)`,
     `- duration: ${day(event.end?.dateTime).diff(event.start?.dateTime, "minute")} minutes`
   ].join("\n"));
 };
 
 const createNextEvents = async(email: string): Promise<void> => {
-  const events = await google.getEvents(email, day(), day().add(7, "day"));
+  const events = await google.getEvents(email, day(), day().add(Number(env.DAYSTOFECTH), "day"));
```

Comment:
You need to add this new environment variable to the README with a short line of explanation, as is the case for other environment variables.